src/main/java/games/strategy/triplea/oddscalc/OddsCalculatorPanel.java

```diff
@@ -256,25 +241,6 @@ private void setupListeners() {
       updateAttacker(null);
       setWidgetActivation();
     });
-    calculateButton.addMouseMotionListener(new MouseMotionListener() {
-      @Override
-      public void mouseDragged(final MouseEvent e) {}
-
-      @Override
-      public void mouseMoved(final MouseEvent e) {
-        final String memoryAvailable = "<br/>Percentage of memory available: "
-            + String.format("%.2f", (percentageOfFreeMemoryAvailable() * 100)) + "% <br/>Free memory available: "
-            + (freeMemoryAvailable() / (1024 * 1024)) + "MB <br/>Maximum allowed memory: "
```

Comment:
The `freeMemoryAvailable()` method in this class is no longer used and can be removed.

Reply:
87ab122